environment.yml

```diff
@@ -4,55 +4,54 @@ channels:
     - https://conda.anaconda.org/conda-forge
 dependencies:
     - python==3.5.2
+    # Please keep alphabetized
+    - mkl-service=1.1.2
+    - mkl==2017.0.4
     - numpy==1.14.5
     - scipy
-    - mkl==2017.0.4
-    - path.py
-    - python-dateutil
-    - joblib==0.10.3
-    - mako
-    - ipywidgets
-    - flask
-    - matplotlib
-    - pandas
-    - mkl-service=1.1.2
-    # absl-py must be installed via conda rather than pip, because for some
-    # reason depedencies from the pip stanza are not available during the build
-    # phase of other packages in the pip stanza
-    - absl-py
```

Comment:
We don't need absl-py anymore?

Reply:
A "working" dm_control pip install actually fails the first time, then installs absl-py, then tries again and succeeds.